test/dummy/db/schema.rb

```diff
@@ -21,17 +21,20 @@
     t.integer "count", default: 0, null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.index ["action", "name", "tracker_id"], name: "index_billing_usage_counts_on_action_and_name_and_tracker_id", unique: true
```

Comment:
There doesn't seem to be a migration for this change. Do we need one? Or should we drop this line? (I realize this is the schema for the dummy app, just wondering if downstream apps need this change or not.)